### PR TITLE
fix: Add simple retries on network errors

### PR DESCRIPTION
**Pull request by adinhodovic**

### CODE REVIEW

File: celery_progress/static/celery_progress/celery_progress.js [hunk 1]
```diff
@@ -138,11 +139,23 @@ class CeleryProgressBar {
...
+        let attempts = 0;
+        while(!success && attempts < this.maxNetworkRetryAttempts) {
+          try {
+              response = await fetch(this.progressUrl);
+              success = true;
+          } catch (networkError) {
+              error = networkError;
+              this.onNetworkError(this.progressBarElement, this.progressBarMessageElement, "Network Error");
+              attempts++;
+              await new Promise(r => setTimeout(r, 1000));
```
Comment (czue) [inline celery_progress/static/celery_progress/celery_progress.js:153]: sorry my async JS is not so good. How does this end up doing a retry? like how does it know what to call again?

**Review by czue — COMMENTED** [review]
thanks! one (maybe dumb) question about how this works...

File: celery_progress/static/celery_progress/celery_progress.js (hunk 1 above)
Comment (czue) [reply celery_progress/static/celery_progress/celery_progress.js:153]: Ah right, duh. And this is essentially the equivalent of an async sleep?

Edit: Looks like yes https://stackoverflow.com/a/39914235/8207

**Review by czue — APPROVED** [review]

File: celery_progress/static/celery_progress/celery_progress.js (hunk 1 above)
Comment (adinhodovic) [reply celery_progress/static/celery_progress/celery_progress.js:153]: Oh yes, this was just not to spam the backend with requests every miliseconds or so.

**Comment by czue** [comment]
Done! Sorry about the delay. https://github.com/czue/celery-progress/releases/tag/0.3

**Pull request merged**